src/components/date/date.md

```diff
@@ -0,0 +1,64 @@
+## Purpose
```

Comment:
TODO: Move wordy documentation to Notion for a better collaborative environment.